addons/account/models/account_move.py

```diff
@@ -122,6 +122,12 @@ def create(self, vals):
 
     @api.multi
     def write(self, vals):
+        values = []
```

Comment:
remove the hunk in the write() method: this feature should only be triggered via the onchange + refactore the onchange method since we need only one now